README.md

```diff
@@ -29,6 +29,7 @@ The following fields can be used for assets (in the [`Asset Object`](https://git
 | file:header_size     | integer                                 | The header [size](#sizes) of the file, specified in bytes.   |
 | file:size            | integer                                 | The file [size](#sizes), specified in bytes.                 |
 | file:values          | \[[Mapping Object](#mapping-object)\]   | Lists the value that are in the file and describes their meaning. See the [Mapping Object](#mapping-object) chapter for an example. If given, at least one array element is required. |
+| file:rel_path        | string                                  | [Relative path](#relative-path) of the asset                 |
```

Comment:
I'm just wondering whether local_path would be a good name? It would cover local path on the computer after download and local path in the archive...